app/controllers/projects/samples/transfers_controller.rb

```diff
@@ -4,7 +4,6 @@ module Projects
   module Samples
     # Controller actions for Project Samples Transfer
     class TransfersController < ApplicationController
```

Comment:
**nitpick (blocking):** Since you removed the before action here, I believe you need to update this to inherit from `Projects::ApplicationController` instead of just `ApplicationController`